src/views/menus/AccountMenu/AccountMenu.tsx

```
@@ -193,252 +198,255 @@ export const AccountMenu = () => {
+                ].filter(isTruthy)}
+            slotBottomContent={< MobileDownloadLinks withBadges />}
+            align="end"
+            sideOffset={16}
+        >
+            {walletIcon}
+            {!isTablet && <$Address>{truncateAddress(dydxAddress)}</$Address>}
+        </$DropdownMenu >
+    );
 };
 
 const DydxDerivedAddress = ({
```

ESLint

[error] 'DydxDerivedAddress' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)